pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools>=57.4",
```

Comment:
Introduces the env var [SETUPTOOLS_EXT_SUFFIX](https://github.com/pypa/setuptools/blob/v60.9.0/setuptools/command/build_ext.py#L106-L112) ([ref](https://github.com/pypa/setuptools/issues/2722))